### ISSUE TITLE
[DetailsList][Accessibility]how to navigate to the second link of a column cell

### ISSUE THREAD

**Issue by LinglingZhangMicrosoft** [microsoft/fluentui#22558]
### Environment Information

- **Package version(s)**: (fill this out)
- **Browser and OS versions**: (fill this out if relevant)

### Describe the issue:

one column cell has more than one links like this, can't navigate to the second link with keyboard
![image](https://user-images.githubusercontent.com/98578535/164180064-8f98e90d-dae1-4eb7-87f9-e2f8db2fc796.png)

#### Actual behavior:
right arrow key could visit the first link. but i don't know how to visit the second link

#### Expected behavior:
how to visit the second link with keyboard?

### If applicable, please provide a codepen repro:
https://developer.microsoft.com/en-us/fluentui#/controls/web/detailslist/customitemcolumns, this example with a little change to modify the name column to 
![image](https://user-images.githubusercontent.com/98578535/164180980-688170d5-d62d-490a-a17e-44db88e910dd.png)

### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? (Yes, No)
No
Requested priority: (Blocking, High, Normal, Low)
Normal
Products/sites affected: (if applicable)

**Comment by gouttierre** [comment]
@LinglingZhangMicrosoft - Thanks for filing this issue with us. Please, retest your issue as i followed the repro steps and was unable to reproduce the issue stated. I was able to focus on the second link using the keyboard navigation.

**Comment by LinglingZhangMicrosoft** [comment]
can't repro now. thanks for checking.
